### PR TITLE
Tidied printing

### PR DESCRIPTION
**Pull request by dmalan**

### CODE REVIEW

File: submit50 [hunk 1]
```diff
@@ -404,6 +299,138 @@ run.GIT_DIR = tempfile.mkdtemp()
...
+
+    # add a tag reference
+    run("git tag --force '{}'".format(tag))
+    run("git push --force origin 'refs/tags/{}'".format(tag), password=password)
+
+    # stop the spinner
+    spin(False)
+
+    # successful submission
+    cprint("Submitted {}! See https://cs50.me/submissions/{}.".format(problem, branch), "green")
```
Comment (dmalan) [inline submit50]: @brianyu28 any concerns with outputting this more direct URL?

**Comment by dmalan** [comment]
In addition to some other minor tweaks, this PR adds `cprint`, which simplifies colorized printing and also ensures long lines (like the academic-honesty prompt) break on words, not chars.

File: submit50 (hunk 1 above)
Comment (brianyu28) [inline submit50]: Should be fine!

**Comment by brianyu28** [comment]
All looks good!

**Pull request merged**